### ISSUE TITLE
Move configs from Configuration.ts into a separate config.json file

### ISSUE THREAD

**Issue by jondubois** [Leasehold/lisk-dex-ui#5]
Users should be able to modify the configs without having to rebuild the projects.

**Comment by jondubois** [comment]
After moving the config from `Configuration.ts` to a new `config.json` file, we can remove the type definitions. If TypeScript is not used too much, we can remove it.

**Comment by sacOO7** [comment]
I have removed type definitions for this, as a part of new PR, you can take a look at it. And you can close this issue, once the PR is merged.

**Comment by sacOO7** [comment]
I think we can close this issue since the typescript PR is merged.
